lib/adhearsion/call_controller.rb

```diff
@@ -108,8 +108,9 @@ def exec(controller = self)
 
     def bg_exec(completion_callback = nil)
       Celluloid::ThreadHandle.new(Celluloid.actor_system) do
-        catching_standard_errors do
+        begin
           exec_with_callback completion_callback
+        rescue StandardError
```

Comment:
True, it could happen. `exec_with_callback` and `exec` are short and simple but the optional `completion_callback` wouldn't be covered either. Any suggestions?

Reply:
What about removing the deeper exception catcher/logger, and leaving this one in place? Any downside?

Reply:
I guess we'd keep the `on_error` call and the reraise, removing just the trigger. I've checked and `execute!` is not called from elsewhere. It's also marked (but is not really) private so this should be fine. I'll give it a try.